lib/pbench/server/database/models/users.py

```diff
@@ -5,9 +5,21 @@
 from flask_bcrypt import generate_password_hash
 from sqlalchemy import Column, DateTime, Enum, Integer, LargeBinary, String
 from sqlalchemy.orm import relationship, validates
+from sqlalchemy.orm.exc import NoResultFound
 
 from pbench.server.database.database import Database
-from pbench.server.database.models.active_tokens import ActiveTokens  # noqa[F401]
+
+
+class AdminUserDeletion(Exception):
```

Comment:
This exception does not appear to be related to the users model.  This seems to be only used by the `user_api` module.  Perhaps it should be moved there?

Reply:
Tracked in issue #2334.